sycl/test-e2e/KernelAndProgram/kernel-bundle-merge-options.cpp

```diff
@@ -1,21 +1,24 @@
 // REQUIRES: gpu
-// RUN: %{build} -o %t.out %debug_option
+// RUN: %{build} -o %t.out -g
 // RUN: env SYCL_UR_TRACE=2 %{run} %t.out | FileCheck %s
 // UNSUPPORTED: hip
 
-// Debug option -g is not passed to device code compiler when CL-style driver
```

Comment:
Another concern here is about those who actually uses CL-style driver. Does it mean that we should release note that `/DEBUG` **does not** enable special debug flow in backend compiler?

It looks like that instead of fixing a problem we are simply updating the test so that it ignores the problematic path

Reply:
On CL I don't believe `/DEBUG` is the right flag. The equivalent to `-g` is `/Zi`, which IS getting correctly translated to `-g` when we call the driver.  
IIRC, `/DEBUG` outputs one of those .pdb files, and is used by the Linker, no?

Reply:
I believe the problem is that the `debug_option` in lit.cf.py needs to have both flags.  Have updated the PR.

Reply:
Looking into MSVC documentation for [`/Zi`](https://learn.microsoft.com/en-us/cpp/build/reference/z7-zi-zi-debug-information-format?view=msvc-170) and [`/DEBUG`](https://learn.microsoft.com/en-us/cpp/build/reference/debug-generate-debug-info?view=msvc-170) I agree: `/DEBUG` is a linker flag, it doesn't generate debug info by itself, but is simply used to create a "combined" `.pdb` for all linked files.
Therefore, the assumption that `/DEBUG` should lead to `-g` being passed to device compiler is incorrect.

Note: `/Zi` implies `/DEBUG` as documentation says